jenkins/ci.suse.de/openstack-submit-project.yaml

```diff
@@ -42,43 +42,43 @@
               ;;
           esac
 
-          export BS=IBS
-          OSC_OBS="osc -A https://api.opensuse.org"
-          OSC_IBS="osc -A https://api.suse.de"
-          export OSC="$OSC_IBS"
+          export bs=IBS
+          osc_obs="osc -A https://api.opensuse.org"
+          osc_ibs="osc -A https://api.suse.de"
+          export osc="$osc_ibs"
```

Comment:
if this is an internal variable - why did you add export in the 1st commit here?

Reply:
good point - it was in the original code (export OSC ; export BS)
will fix it